subsystem/kubeapi_test.go

```diff
@@ -1524,6 +1526,12 @@ var _ = Describe("[kube-api]cluster installation", func() {
 			return aci.Status.DebugInfo.LogsURL
 		}, "30s", "10s").ShouldNot(Equal(""))
 
+		By("Verify Agent labels")
+		Eventually(func() map[string]string {
```

Comment:
just making sure but is the order guaranteed?

Reply:
It should be. Do you prefer that I set only one label?

Reply:
It looks like `Equal` is using `reflect.DeepEqual` so it should be fine https://www.geeksforgeeks.org/comparing-maps-in-golang/